src/wiki/help-pages/Change-Themes.md

```diff
@@ -0,0 +1,135 @@
+---
+eleventyNavigation:
+  key: Change Themes
+  parent: Help pages
+--- 
+
+# How to change your Themes in Prism Launcher
+
+Themes can help you spice up your launcher appearance
+
+Prism Launcher 6.0 will add support for multiple custom themes at once
+
+***This section will be updated accordingly***
+
+Some themes are only on discord for now, but we will try to transfer them to our repository.
+
+Prism Launcher use Qt stylesheet, to make themes please look at [Qt Documentation](https://doc.qt.io/qt-6/stylesheet-reference.html).
```

Comment:
I will change this line a bit tomorrow probably actually idk

Reply:
Ping me when you're done with this. I'll merge both PRs.